### PR TITLE
Us04 04 banner and text cours

### PR DESCRIPTION
**Pull request by NEL45**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]
ok haras is changed because bad commit for Lucy :)

File: src/View/User/course.html.twig [hunk 1]
```diff
@@ -3,5 +3,6 @@
     <link rel="stylesheet" href="/assets/css/course.css">
 {% endblock %}
 {% block content %}
+    {% include 'Includes/_banner.html.twig' with {image: 'collector.webp', alt: 'desesperate horse', title: 'COURS'} %}
     {% include "User/Includes/_course_tab.html.twig" %}
```
Comment (LucyMerlier) [inline src/View/User/course.html.twig]: use single quotes to be coherent

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by Loic-Code — DISMISSED** [review]

**Review by LucyMerlier — DISMISSED** [review]

**Review by Wowlfy — APPROVED** [review]
bien

**Review by Wowlfy — APPROVED** [review]
bien

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**